modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEUploadField.testcase

```diff
@@ -26,6 +26,46 @@ definition {
 		}
 	}
 
+	@description = "This is a stubbed description"
+	@priority = "5"
+
+	test {
```

Comment:
@guivilarim, where is the test name??

Reply:
@guivilarim, have you had the chance to see this comment? Can you please edit this commit to make it clearer to the reviewer?